app/src/main/java/com/amaze/filemanager/filesystem/files/FileUtils.java

```diff
@@ -579,6 +579,8 @@ public static boolean copyToClipboard(Context context, String text) {
   public static String[] getPathsInPath(String path) {
     if (path.endsWith("/")) path = path.substring(0, path.length() - 1);
 
+    path = path.trim();
```

Comment:
I don't think this will fix the issue. The issue is it can't find any `/` in the path at line 586. So I think moving `trim` method in while loop and a check if `lastIndexOf` is -1 then break out of loop (for safety) should fix the issue.